pkg/apis/monitoring/v1beta1/conversion_to.go

```diff
@@ -31,13 +31,14 @@ func convertRouteTo(in *Route) (*v1alpha1.Route, error) {
 	}
 
 	out := &v1alpha1.Route{
-		Receiver:          in.Receiver,
-		GroupBy:           in.GroupBy,
-		GroupWait:         in.GroupWait,
-		GroupInterval:     in.GroupInterval,
-		RepeatInterval:    in.RepeatInterval,
-		Matchers:          convertMatchersTo(in.Matchers),
-		MuteTimeIntervals: in.MuteTimeIntervals,
+		Receiver:            in.Receiver,
+		GroupBy:             in.GroupBy,
+		GroupWait:           in.GroupWait,
+		GroupInterval:       in.GroupInterval,
+		RepeatInterval:      in.RepeatInterval,
+		Matchers:            convertMatchersTo(in.Matchers),
+		MuteTimeIntervals:   in.MuteTimeIntervals,
+		ActiveTimeIntervals: in.ActiveTimeIntervals,
```

Comment:
Dont' we need a similar change in conversion_from.go?

Reply:
Thanks, updated `conversion_from.go` accordingly